ocw_data_parser/ocw_data_parser.py

```diff
@@ -467,7 +467,12 @@ def update_s3_content(self, upload=None, update_pages=True, update_media=True, m
                     filename = uid + "_" + safe_get(file, "id")
                     if not get_binary_data(file):
                         log.error(
-                            "Could not load binary data for file: %s", filename)
+                            "Could not load binary data for file %s in json file %s for course %s",
+                            filename,
+                            safe_get(file, "actual_file_name"),
+                            safe_get(self.master_json, "short_url")
+                        )
+                        continue
```

Comment:
This seems to be a change of behavior since the missing `d` would have caused an error a few lines down. Do we  want to continue processing other files here? Just double checking

Reply:
I think we do want to continue. It does not make sense to not import the whole course if one file has a data issue